setup.py

```diff
@@ -21,27 +21,39 @@
 
 
 install_requires = [
+    #
+    # The core 'install_requires' should only be things
+    # which are needed for the main editor to function.
+    #
     "PyQt5==5.13.2"
     + ';"arm" not in platform_machine and "aarch" not in platform_machine',
     "QScintilla==2.11.3"
     + ';"arm" not in platform_machine and "aarch" not in platform_machine',
     "PyQtChart==5.13.1"
     + ';"arm" not in platform_machine and "aarch" not in platform_machine',
+    #
+    # FIXME: Maybe should be in a mode?
+    # qtconsole, pyserial
+    #
+    "qtconsole==4.7.4",
+    "pyserial==3.4",
     # `flake8` is actually a testing/packaging dependency that, among other
     # packages, brings in `pycodestyle` and `pyflakes` which are runtime
     # dependencies. For the sake of "locality", it is being declared here,
     # though. Regarding these packages' versions, please refer to:
     # http://flake8.pycqa.org/en/latest/faq.html#why-does-flake8-use-ranges-for-its-dependencies
     "flake8 >= 3.8.3",
-    "pyserial==3.4",
-    "qtconsole==4.7.4",
-    "pgzero==1.2",
     "appdirs>=1.4.3",
     "semver>=2.8.0",
-    "nudatus>=0.0.3",
     'black>=19.10b0;python_version > "3.5"',
```

Comment:
Technically black is optional, in case you'd prefer to move that to the virtual environment.